### PR TITLE
fix bot

### PR DESCRIPTION
**Pull request by jeremyheaton**
# All Submissions

 [x] Did you have fun?

## What did you build

-------------------------------------

Issue Number: N/A

### CODE REVIEW

File: FoodFite/Models/UserProfile.cs [hunk 1]
```diff
@@ -13,11 +13,21 @@ namespace FoodFite.Models
...
     {
         public string Name { get; set; }
+        public string Opponent { get; set; }
+        public string Weapon { get; set; }
         public List<Item> Inventory { get; set;}
         public double Health { get; set;}
         public Protection Clothes { get; set; }
+        public Dictionary<string, Food> FoodMap { get; set; }
+        public List<Food> FoodInventory{get; set;}
+        public Protection Protection{get; set;}
```
Comment (dlambright03) [inline FoodFite/Models/UserProfile.cs]: This looks to be the same as clothes

File: FoodFite/Models/UserProfile.cs [hunk 2]
```diff
@@ -13,11 +13,21 @@ namespace FoodFite.Models
     public class UserProfile
     {
         public string Name { get; set; }
+        public string Opponent { get; set; }
```
Comment (dlambright03) [inline FoodFite/Models/UserProfile.cs:16]: Why is Opponent and Weapon here?

File: FoodFite/Models/UserProfile.cs [hunk 3]
```diff
@@ -13,11 +13,21 @@ namespace FoodFite.Models
...
+        public string Weapon { get; set; }
         public List<Item> Inventory { get; set;}
         public double Health { get; set;}
         public Protection Clothes { get; set; }
+        public Dictionary<string, Food> FoodMap { get; set; }
```
Comment (dlambright03) [inline FoodFite/Models/UserProfile.cs:21]: What is the use of this and FoodInventory? Inventory and ListFood() should cover these

**Review by dlambright03 — CHANGES_REQUESTED** [review]

File: FoodFite/Models/UserProfile.cs (hunk 2 above)
Comment (jeremyheaton) [reply FoodFite/Models/UserProfile.cs:16]: it's needed right now to just get the bot running. Those are values that need to persist. We need to leave this here until we find a better way to persist it.

File: FoodFite/Models/UserProfile.cs (hunk 1 above)
Comment (jeremyheaton) [reply FoodFite/Models/UserProfile.cs]: yeah this was due to us both working the model at the same time. I'll update this to use your fields.

File: FoodFite/Models/UserProfile.cs (hunk 3 above)
Comment (jeremyheaton) [reply FoodFite/Models/UserProfile.cs:21]: It's a look up for when a food is selected. All of our inputs are text, so if I select a grapes, I need a way to look up my grapes object. This also allows merging of foods for the future.

File: FoodFite/Models/UserProfile.cs (hunk 2 above)
Comment (dlambright03) [reply FoodFite/Models/UserProfile.cs:16]: ok

File: FoodFite/Models/UserProfile.cs (hunk 3 above)
Comment (dlambright03) [reply FoodFite/Models/UserProfile.cs:21]: FoodMap should probably live elsewhere but I guess this works for now

**Review by dlambright03 — APPROVED** [review]

**Pull request merged**